proto/kvrpcpb.proto

```diff
@@ -655,3 +655,106 @@ message ReadIndexResponse{
     uint64 read_index = 2;
 }
 
+enum VerOp {
+    VerPut = 0;
+    VerDel = 1;
+}
+
+message VerMutation {
+    VerOp op = 1;
+    bytes key = 2;
+    bytes value = 3;
+}
+
+message VerValue {
+    bytes value = 1;
+    uint64 version = 2;
+}
+
+message VerKeyError {
+    string error = 1; 
+}
+
+message VerKvPair {
+    VerKeyError error = 1;
+    bytes key = 2;
+    repeated VerValue values = 3;
+}
+
+message VerGetRequest {
+    Context context = 1;
+    bytes key = 2;
+    string cf = 3; 
+    uint64 start_version = 4; // start_version == 0 means without start version
+}
+
+message VerGetResponse {
+    errorpb.Error region_error = 1;
+    VerKeyError error = 2;
+    repeated VerValue value = 3;
+    bool not_found = 4;
+}
+
+message VerBatchGetRequest {
+    Context context = 1;
+    repeated bytes key = 2;
+    string cf = 3;
```

Comment:
I think we should support having different keys belonging to different cf?

Reply:
I think keeping it simple is better because getting multiple keys from different cf can implement by calling this function multiple times.

Reply:
And I guess it is not a high-frequency operation?

Reply:
for frequency, we mainly use batch operation for performance.